### PR TITLE
Update to reflect latest API changes in Corda

### PR DESCRIPTION
**Pull request by vkolomeyko**

### CODE REVIEW

File: java-source/src/main/java/com/example/flow/ExampleFlow.java [hunk 1]
```diff
@@ -84,7 +84,7 @@ public ProgressTracker getProgressTracker() {
         @Override
         public SignedTransaction call() throws FlowException {
             // Obtain a reference to the notary we want to use.
-            final Party notary = getServiceHub().getNetworkMapCache().getNotaryNodes().get(0).getNotaryIdentity();
+            final Party notary = getServiceHub().getNetworkMapCache().getAnyNotary();
```
Comment (kasiastreich) [inline java-source/src/main/java/com/example/flow/ExampleFlow.java]: Change it to `getNotary()`, Andrius is working now on removing `getAnyNotary` function.

File: kotlin-source/src/main/kotlin/com/example/flow/ExampleFlow.kt [hunk 2]
```diff
@@ -73,7 +73,7 @@ object ExampleFlow {
         @Suspendable
         override fun call(): SignedTransaction {
             // Obtain a reference to the notary we want to use.
-            val notary = serviceHub.networkMapCache.notaryNodes.single().notaryIdentity
+            val notary = serviceHub.networkMapCache.getAnyNotary()
```
Comment (kasiastreich) [inline kotlin-source/src/main/kotlin/com/example/flow/ExampleFlow.kt]: Same

File: java-source/src/main/java/com/example/flow/ExampleFlow.java (hunk 1 above)
Comment (vkolomeyko) [reply java-source/src/main/java/com/example/flow/ExampleFlow.java]: But `getNotary()` is defined as:
`fun getNotary(name: CordaX500Name): Party?`
which `name` are we meant to pass in?

**Comment by kchalkias** [comment]
We have a similar example in [FlowCookBook.java](https://github.com/corda/corda/blob/master/docs/source/example-code/src/main/kotlin/net/corda/docs/FlowCookbook.kt). I guess you could comment out the different strategies as well.

**Comment by kasiastreich** [comment]
https://github.com/corda/corda/pull/1573 here is the PR with notary selection cleanup

File: java-source/src/main/java/com/example/flow/ExampleFlow.java (hunk 1 above)
Comment (kasiastreich) [reply java-source/src/main/java/com/example/flow/ExampleFlow.java]: Ah, sorry I forgot about that, you could choose the first one from notaries list. However it should be commented similarly to `FlowCookbook` on Andrius PR.

**Pull request closed without merging**